src/elements/kw-share-feed/kw-share-feed.html

```diff
@@ -58,14 +65,29 @@
             }
         </style>
         <div class="shell" id="shell">
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
-            <kw-share-card share="[[shellShare]]"></kw-share-card>
+            <!--<kw-share-card share="[[shellShare]]"></kw-share-card>
+            <kw-share-card share="[[shellShare]]"></kw-share-card>-->
+            <kw-share-shell share="[[shellShare]]" slotName="share-card">
+                <div slot="share-card">
+                    <kw-share-card share="[[shellShare]]"></kw-share-card>
```

Comment:
Don't put the share card inside the shell, put the shell inside the card, as we designed